src/app/layout.tsx

```diff
@@ -29,7 +29,7 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body className={twMerge(inter.className, "bg-grey-dark text-light-300")}>
-        <div>
+        <div className="min-h-screen flex flex-col justify-between">
```

Comment:
Needed for the 404 page. I've tested and this shouldn't break anything. cc @dohaki